packages/py/polywrap-msgpack/polywrap_msgpack/__init__.py

```diff
@@ -21,8 +21,16 @@ def sanitize(value: Any) -> Any:
         array: List[Any] = value
         return [sanitize(a) for a in array]
     elif type(value) is tuple:
-        array: List[Any] = list(*value)
-        return [sanitize(a) for a in array]
+        array: List[Any] = list(value)
+        return sanitize(array)
+        ##previous implementation below:
+        #array: List[Any] = list(*value)
+        #return [sanitize(a) for a in array]
```

Comment:
Remove this

Reply:
wanted to make sure the new implementation was correct before getting rid of the comment
what was making this not work before, i think, was that we were placing the sanitized list within another list, therefore nestingthem uncessarily